amdpatch.sh

```diff
@@ -2,4 +2,4 @@
 
 ROCM_PATH=$(hipconfig --rocmpath)
 
-sudo patch $ROCM_PATH/lib/llvm/lib/clang/18/include/__clang_hip_cmath.h ./patches/amd.patch
\ No newline at end of file
+sudo patch $ROCM_PATH/lib/llvm/lib/clang/*/include/__clang_hip_cmath.h ./patches/amd.patch
```

Comment:
What happens if there's multiple versions installed here? Will it return the first one?

Reply:
I think there shouldn't be more than one version, though I can just make it check rocm version and patch clang 17 for rocm 6.1 and clang 18 for 6.2 if that's better

Reply:
Nvm, I'm too stupid for that. It should be fine as is anyway